x-pack/plugins/apm/public/components/app/correlations/error_correlations.tsx

```diff
@@ -170,6 +170,7 @@ export function ErrorCorrelations({ onClose }: Props) {
               'xpack.apm.correlations.error.percentageColumnName',
               { defaultMessage: '% of failed transactions' }
             )}
+            // @ts-expect-error: this file to be removed
```

Comment:
Maybe we can remove this file as part of this PR already?

Reply:
Deleted here https://github.com/elastic/kibana/pull/108441/commits/e70878c6ca042f7e7f97bc91f94443cc492200ba